include/mqt-core/dd/GateMatrixDefinitions.hpp

```diff
@@ -174,4 +140,15 @@ inline TwoQubitGateMatrix xxPlusYYMat(const fp theta, const fp beta = 0.) {
        {0, {-sinBeta * sinTheta, -cosBeta * sinTheta}, cosTheta, 0},
        {0, 0, 0, 1}}};
 }
+
+inline GateMatrix opToSingleGateMatrix(qc::OpType t,
+                                       const std::vector<fp>& params = {}) {
+  return MATS_GENERATORS.at(t)(params);
+}
+
+inline TwoQubitGateMatrix
+opToTwoQubitGateMatrix(qc::OpType t, const std::vector<fp>& params = {}) {
+  return TWO_GATE_MATS_GENERATORS.at(t)(params);
+}
```

Comment:
Also just slightly more precise naming and a small `const` addition.
```suggestion
inline GateMatrix opToSingleQubitGateMatrix(const qc::OpType t,
                                       const std::vector<fp>& params = {}) {
  return MATS_GENERATORS.at(t)(params);
}

inline TwoQubitGateMatrix
opToTwoQubitGateMatrix(const qc::OpType t, const std::vector<fp>& params = {}) {
  return TWO_GATE_MATS_GENERATORS.at(t)(params);
}
```